src/shared/cloudformation/templateRegistry.ts

```diff
@@ -34,7 +34,7 @@ export class CloudFormationTemplateRegistry extends WatchedFiles<CloudFormation.
         try {
             template = await CloudFormation.load(path)
         } catch (e) {
-            await updateYamlSchemasArray(path, 'none', this.config)
+            await updateYamlSchemasArray(path, 'none', { config: this.config, skipExtensionLoad: !!this.config })
```

Comment:
We are now using `testOverrides` in non-test code. I guess it should be named `opt`.